examples/html-ep/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "vite build"
   },
   "dependencies": {
-    "@walletconnect/ethereum-provider": "2.17.4-canary-ak-1"
+    "@walletconnect/ethereum-provider": "2.18.0-canary-ak-8"
```

Comment:
Should we keep them canary? Did we release new ep?

Reply:
Lets keep them as canary, will update as new Vs come out